packages/playground/src/components/node_details.vue

```diff
@@ -0,0 +1,308 @@
+            <v-list-item>
+              <v-list-item-content style="display: flex; justify-content: space-between; align-items: center">
+                <v-list-item-title>Certification type</v-list-item-title>
+                {{ item.value.certificationType }}
+              </v-list-item-content>
+            </v-list-item>
+            <v-divider></v-divider>
+            <v-list-item>
+              <v-list-item-content style="display: flex; justify-content: space-between; align-items: center">
+                <v-list-item-title>Public Ips</v-list-item-title>
+                {{ publicIps }}
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-card>
+      </v-col>
+
+      <v-col v-if="item.value.num_gpu > 0" cols="getColSize" style="min-width: 600px">
+        <v-card flat color="transparent" tag="div">
+          <!-- Title -->
+          <v-card color="primary" class="d-flex justify-center items-center mt-3 pa-3 text-center">
+            <v-icon size="30" class="pr-3">mdi-expansion-card-variant</v-icon>
+            <v-card-title class="pa-0" color="white"
+              >GPU Card{{ item.value.num_gpu > 1 ? "s" : "" }} details</v-card-title
+            >
+          </v-card>
+
+          <!-- Details -->
+          <v-row v-if="loading">
+            <v-col cols="12" class="text-center pt-12">
+              <v-progress-circular indeterminate size="64" color="primary"></v-progress-circular>
+            </v-col>
+          </v-row>
+          <v-row v-else-if="gpuItem && gpuItem.length !== 0">
+            <v-col cols="12">
+              <v-list>
+                <v-list-item>
+                  <v-list-item-content style="display: flex; justify-content: space-between; align-items: center">
+                    <v-tooltip top nudge-bottom="30">
+                      <template v-slot:activator="{ isActive, props }">
+                        <v-list-item-title v-bind="props" v-on="isActive"
+                          >Card ID
+                          <v-chip
+                            lose-icon="mdi-delete"
+                            :color="gpuItem[0].contract ? 'warning' : 'success'"
+                            x-small
+                            class="mb-1 ml-3"
+                            >{{ gpuItem[0].contract ? "Reserved" : "Available" }}</v-chip
+                          >
+                        </v-list-item-title>
+                      </template>
+                      <span>Card id that's used in a deployment</span>
+                    </v-tooltip>
+                    <v-col class="pa-0 d-flex justify-end align-center pl-2">
+                      <v-select
+                        style="max-width: 250px"
+                        v-if="gpuItem.length > 1"
+                        append-outer-icon="mdi-content-copy"
+                        hide-details
+                        dense
+                        v-model="gpuItem[0].id"
```

Comment:
we should show the rest gpu cards as well. please check the old dashboard implementation